### DISCUSSION TITLE
Seq tracing not being shown as hierarchy for me

### DISCUSSION THREAD

**Discussion by nabeel-servcorp** [datalust/seq-tickets#2131] [Help] [answered]
I followed the instructions available on the official [SerilogTracing](**[url](https://github.com/serilog-tracing/serilog-tracing/)**) page. I can see the HTTP traces appear in my Seq instance but they are not in hierarchical form (as the examples show).

What is the requirement for that fomr?

**Comment by nblumhardt** [comment]
Hi! Are you using Seq 2024.1? Earlier versions won't show hierarchical traces.

If you are, it might help to send some additional info (redacted screenshots, Serilog + SerilogTracing configuration). Feel free to send those details to support@datalust.co if you prefer.

**Comment by nabeel-servcorp** [comment, marked as answer]
The issue is fixed. The key was to click on 'Trace' menu option on top and then 'Show'.

**Reply by nblumhardt** [reply]
Great! How does it look? 😎

**Reply by nabeel-servcorp** [reply]
As expected, in a hierarchy.
